man/systemd.network.xml

```diff
@@ -247,6 +247,12 @@
         <listitem>
           <para> A boolean. Enables or disables the change the MULTICAST flag on the device.</para>
```

Comment:
This does not parse. Please fix it too.

Reply:
https://books.google.co.in/books?id=MTibAgAAQBAJ&pg=PT556&lpg=PT556&dq=IFF_ALLMULTI&source=bl&ots=ITFWupPIGO&sig=P7J_zzqF98bSv6aMm8qzwine_Yo&hl=en&sa=X&ved=0ahUKEwiq-J3jh7DbAhUCvrwKHR4tBTQ4ChDoAQhNMAc#v=onepage&q=IFF_ALLMULTI&f=false

Reply:
Right. But the problem is that the sentence in the man page above is grammatically incorrect, and in effect it's very unclear.

Reply:
please suggest

Reply:
I am not native, but I think the minimal way to fix this is just omitting 'the change'.

Reply:
@yuwata I pushed a new  man. Please take a look

Reply:
Hm? It seems not updated...